src/bci_build/package/basecontainers.py

```diff
@@ -330,6 +334,56 @@ def _get_minimal_kwargs(os_version: OsVersion):
     )
 
 
+# SL Micro 6.0 GA kernel container devel
+_SLM60_BASEURL = "https://api.opensuse.org/public/build/SUSE:ALP:Source:Standard:Core:1.0:Build/standard/"
+_SLM60_KERNEL_PACKAGES = [
+    "x86_64/patchinfo.ga/kernel-devel-6.4.0-17.1.noarch.rpm",
+    "x86_64/patchinfo.ga/kernel-macros-6.4.0-17.1.noarch.rpm",
+    "x86_64/patchinfo.ga/kernel-syms-6.4.0-17.1.x86_64.rpm",
+    "x86_64/patchinfo.ga/kernel-default-devel-6.4.0-17.1.x86_64.rpm",
+    "aarch64/patchinfo.ga/kernel-syms-6.4.0-17.1.aarch64.rpm",
+    "aarch64/patchinfo.ga/kernel-default-devel-6.4.0-17.1.aarch64.rpm",
+    "aarch64/patchinfo.ga/kernel-64kb-devel-6.4.0-17.1.aarch64.rpm",
+    "s390x/patchinfo.ga/kernel-syms-6.4.0-17.1.s390x.rpm",
+    "s390x/patchinfo.ga/kernel-default-devel-6.4.0-17.1.s390x.rpm",
+]
+KERNEL_MODULE_CONTAINERS.append(
+    OsContainer(
+        name="slm60-kernel-module-devel",
+        pretty_name="SUSE Linux Micro 6.0 Kernel module development",
+        logo_url="https://opensource.suse.com/bci/SLE_BCI_logomark_green.svg",
+        os_version=OsVersion.SLE16_0,
+        supported_until=_SUPPORTED_UNTIL_SLE.get(os_version),
+        is_latest=OsVersion.SLE16_0 in CAN_BE_LATEST_OS_VERSION,
+        package_list=(
+            [
+                "gcc-build",
+                "kmod",
+                "make",
+                "patch",
+                "gawk",
+                "pesign-obs-integration",
+                "dwarves",
+                "libelf-devel",
+                *OsVersion.SLE16_0.release_package_names,
+            ]
+        ),
+        exclusive_arch=[Arch.X86_64, Arch.S390X, Arch.AARCH64],
+        extra_files={"_constraints": generate_disk_size_constraints(8)},
+        custom_end=_get_asset_script(_SLM60_BASEURL, _SLM60_KERNEL_PACKAGES)
+        + textwrap.dedent(
+            f"""
+            {DOCKERFILE_RUN} \\
+                [ $(LC_ALL=C rpm --checksig -v *rpm | \\
+                    grep -c -E "^ *V3.*key ID 09d9ea69:") = {len(_SLM60_KERNEL_PACKAGES)} ] \\
```

Comment:
nit: should this key number be a variable alongside `_SLM60_KERNEL_PACKAGES` for future updates?

Reply:
It's the signing key, we're not going to change that.. but I don't care, I can make it a variable if that's wanted.